### PR TITLE
Update connector-acceptance-tests-reference.md

### PR DESCRIPTION
**Pull request by evantahler**
This information was lost in https://github.com/airbytehq/airbyte/pull/21846 and previously updated in https://github.com/airbytehq/airbyte/pull/22167, and is now un-mergable.

### CODE REVIEW

File: docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md [hunk 1]
```diff
@@ -362,3 +362,36 @@ acceptance_tests:
 
 We cache discovered catalogs by default for performance and reuse the same discovered catalog through all tests.
 You can disable this behavior by setting `cached_discovered_catalog: False` at the root of the configuration.
+
+## Additional Checks
+
+While not nesciacily related to SAT testing, Airbyte employs a number of additional checks which run on connector Pull Requests which check the following items:
```
Comment (flash1293) [inline docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md:368]: Nit: Mentioning SAT, should either be “connector acceptance testing” or maybe just omitting the first half of the sentence

**Review by flash1293 — APPROVED** [review]
Sorry, seems like a bad merge. Left one nit but no re-review required

**Comment by octavia-approvington** [comment]
This code is at peace with itself <br/> imagine karate kid approving

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**